MediaBrowser.Api/Images/RemoteImageService.cs

```diff
@@ -265,16 +265,17 @@ private async Task DownloadImage(string url, Guid urlHash, string pointerCachePa
             {
                 Url = url,
                 BufferContent = false
-
             }).ConfigureAwait(false);
-            var ext = result.ContentType.Split('/').Last();
+            var ext = result.ContentType.Split('/')[^1];
 
             var fullCachePath = GetFullCachePath(urlHash + "." + ext);
 
             Directory.CreateDirectory(Path.GetDirectoryName(fullCachePath));
-            using (var stream = result.Content)
+            var stream = result.Content;
+            var filestream = new FileStream(fullCachePath, FileMode.Create, FileAccess.Write, FileShare.Read, IODefaults.FileStreamBufferSize, true);
```

Comment:
`stream` won't be disposed if the `FileStream` constructor throws an exception :/